src/main/resources/steps/ui_common.steps

```diff
@@ -0,0 +1,28 @@
+
+Composite: When Navigate to the SauceDemo website homepage
+Given I am on main application page
+When I wait until element located by `caseSensitiveText(Swag Labs)` appears
+Then `${current-page-url}` is equal to `https://www.saucedemo.com/`
+When I take screenshot
+
+Composite: When I fill $swagGoodUserName,$swagPassword to saucedemo website
```

Comment:
not clear composite step syntax, I would suggest
When I fill SauceDemo login form with `$username` username and `$password` password